src/hotspot/share/classfile/stringTable.cpp

```diff
@@ -597,39 +598,40 @@ void StringTable::verify() {
 
 // Verification and comp
 class VerifyCompStrings : StackObj {
-  GrowableArray<oop>* _oops;
+  static unsigned string_hash(oop const& str) {
+    return java_lang_String::hash_code_noupdate(str);
+  }
+  static bool string_equals(oop const& a, oop const& b) {
+    return java_lang_String::equals(a, b);
+  }
+
+  ResizeableResourceHashtable<oop, bool,
+                              ResourceObj::C_HEAP, mtInternal,
+                              string_hash, string_equals> _table;
  public:
   size_t _errors;
-  VerifyCompStrings(GrowableArray<oop>* oops) : _oops(oops), _errors(0) {}
+  VerifyCompStrings() : _table(unsigned(_items_count + 8 / 8)), _errors(0) {}
```

Comment:
I guess you mean unsigned((_items_count + 8) / 8)?

Reply:
Thanks for the review. I changed it to `_table(unsigned(_items_count / 8) + 1)`